README.md

```diff
@@ -13,7 +13,3 @@ Coverage:  [![Coverage Status](https://coveralls.io/repos/github/gcivil-nyu-org/
 
 ## Wiki:
 https://github.com/gcivil-nyu-org/fall24-monday-team2/wiki
-
-## TODO:
-- Deleting threads should also delete all replies/posts.
-- Delete threads by user before deleting the user
```

Comment:
I believe this Todo item is not done yet, should we have deleted it yet (Delete threads by user before deleting the user)